index.js

```diff
@@ -301,29 +314,34 @@ module.exports = {
       runtime = null,
       eventType = null,
       repoUrl = null,
+      organization = null,
     } = activityParameters;
 
     const repositoryUrl = repoUrl ? repoUrl : remoteFromLocalGitRepoUrl;
+    const {
+      name: repoName,
+      owner: repoOrganization,
+    } = extractGitHubRepoData(repositoryUrl);
 
-    const repoName = module.exports.extractGitHubRepoPath(repositoryUrl);
     const changelog = await module.exports.generateChangelog(
       commitId,
-      service ? service : repoName,
+      service ?? repoName,
       environment
     );
     const activityBody = {
       activity: {
         id: commitId,
-        service: service ? service : repoName,
+        service: service ?? repoName,
```

Comment:
We cannot support this operators in order to be compatible with Node versions older than 14

Reply:
Remember: 
https://github.com/Kelsus/api-spot-package/commit/8b43a9419505a848503206258e85be31e22c3c6b

Reply:
I forgot it !, solving